### PR TITLE
Multiple fixes for manifests

### PR DESCRIPTION
**Pull request by se35710**

### CODE REVIEW

File: boostnote.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "homepage": "https://boostnote.io/",
     "license": "GPL-3.0-or-later",
-    "version": "0.11.7",
-    "url": "https://github.com/BoostIO/boost-releases/releases/download/v0.11.7/BoostnoteSetup.exe#/dl.7z",
-    "hash": "d4df317aabca17cdc62c56deaf934ddc223735f777d5d3db6d1b68f55fd5b6f0",
+    "version": "0.11.8",
+    "url": "https://github.com/BoostIO/boost-releases/releases/download/v0.11.8/BoostnoteSetup.exe",
```
Comment (rasa) [inline boostnote.json]: I think we need to leave the `#/dl.7z` on the URL

File: coretemp.json [hunk 2]
```diff
@@ -20,5 +16,22 @@
             "Core Temp.exe",
             "Core Temp"
         ]
-    ]
+    ],
+    "persist": [
+        "CoreTemp.ini"
```
Comment (rasa) [inline coretemp.json]: If `CoreTemp.ini` doesn't already exist in the .zip file, it needs to be created in a `post_install`

File: boostnote.json (hunk 1 above)
Comment (se35710) [reply boostnote.json]: Ahh, yes. I missed that the autoupdate didn't include the `#/dl.7z` part. I'll push a fix in a few seconds.

File: coretemp.json [hunk 3]
```diff
@@ -20,5 +16,23 @@
...
+    ],
+    "checkver": {
+        "url": "https://www.alcpu.com/CoreTemp/history.html",
+        "re": "Version ([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "http://www.alcpu.com/CoreTemp/php/download.php?id=2#/coretemp32.zip"
```
Comment (rasa) [inline coretemp.json]: The checkver is https, but homepage, and the download URLs are http?

File: yatqa.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
     },
     "checkver": {
         "url": "http://31.172.95.122/yatqa/.update",
```
[the remark this answers is not in the document]
Comment (r15ch13) [reply yatqa.json]: Yes, this is the official update URL. Don't ask me why they use a hardcoded IP.

[the remark this answers is not in the document]
Comment (se35710) [reply yatqa.json]: I trippled checked all urls, including reverse DNS lookups, but only using the IP address works.

**Pull request merged**